pkg/exporter/sumologicexporter/exporter_test.go

```diff
@@ -503,7 +515,8 @@ func TestPushJSONLogsWithAttributeTranslation(t *testing.T) {
 		func(w http.ResponseWriter, req *http.Request) {
 			body := extractBody(t, req)
 			var regex string
-			regex += `{"InstanceType":"wizard","host":"harry-potter","log":"Example log","timestamp":\d{13}}`
+			// Mind that host attribute is not being send in log body
+			regex += `{"InstanceType":"wizard","log":"Example log","timestamp":\d{13}}`
```

Comment:
@pmalek-sumo why not `regex := `?

Reply:
Other places had it this way (mostly to keep the indentation at the same level, then you have it all nicely look as if it was a block of text)